mcs/class/System/System.Net.NetworkInformation/IPInterfaceProperties.cs

```diff
@@ -30,6 +30,7 @@
 using System.Globalization;
 using System.IO;
 using System.Net.Sockets;
+using System.Runtime.CompilerServices;
```

Comment:
Without this the build fails with this error:
System.Net.NetworkInformation/IPInterfaceProperties.cs(295,4): error CS0246: The type or namespace name `MethodImplAttribute' could not be found. Are you missing `System.Runtime.CompilerServices' using directive?